optimum/commands/export/openvino.py

```diff
@@ -253,8 +263,9 @@ def run(self):
                     "all_layers": None if is_int8 else self.args.all_layers,
                     "dataset": self.args.dataset,
                     "num_samples": self.args.num_samples,
-                    "quant_method": QuantizationMethod.AWQ if self.args.awq else None,
+                    "awq": self.args.awq,
```

Comment:
I think originally it looked like this but @echarlaix asked to make AWQ option unified with the rest of the HF ecosystem.

Reply:
yes I'd me in favor of keeping the `quant_method` attribute to keep compatibility with what's done in transformers, I think we can have a awq argument instead of a quant_method though if easier : `OVWeightQuantizationConfig(awq=True)` even though slight preference to something like `OVWeightQuantizationConfig(quant_method="awq")` so that when (if) new quantization methods are added it'll be easier than to have multiple argument for each of them

Reply:
@echarlaix Thanks for the idea. We indeed could keep both `awq=True` and `quant_method="awq"`.